test/cli/test_cli_functions.sh

```diff
@@ -284,10 +283,97 @@ function obc_cycle {
     echo_time "✅  obc cycle is done"
 }
 
+function crd_cycle {  
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
+    local crd_create_array=($(test_noobaa silence crd create &>/dev/stdout | awk '{print $7}' | grep -v "Already Exists" | sed -e 's/[""\\]//g' | sed 's/.$//'))
+    local crd_status_array=($(test_noobaa silence crd status &>/dev/stdout | awk '{print $6}' | grep -v "Exists" | sed -e 's/[""\\]//g' | sed 's/.$//'))
+    echo_time "Starting the crd cycle" 
+
+    test_noobaa timeout crd create
+    if [ $? -ne 0 ]  
+    then           
+        echo_time "❌ crd doesn't exist in the test. exiting."
+        exit 1 
+     else 
+        echo_time "✅ crd already exsists in the test"   
+    fi  
+  
+    echo_time "checking if crd noobaa create command equals to crd status command:"
+    for crd_status_array in ${crd_array[@]}
```

Comment:
Why this loop is here? if we are not populating the arrays here, put this loop next to it.
If we want it after the second create, then we need to populate the array with its output.